### PR TITLE
Fixed image pulling from non-stable external links. Addressed #62.

### PR DESCRIPTION
**Pull request by EmadBinAbid**
Handled #62 

### Summary

An example such sample fix is shown below:

Removed:
```
<img
      src={
          "https://upload.wikimedia.org/wikipedia/commons/7/78/Google_Play_Store_badge_EN.svg"
      }
     alt="Download on the Google Play Store"
/>
```
Added:
```
<img
      src={
          "../../assets/google-play-store-badge.svg"
       }
       alt="Download on the Google Play Store"
/>
```
This pull request is a step towards removing non-stable web links for using SVG files in the code.

### CODE REVIEW

File: assets/download-on-itunes.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      </g>
+      <g id="_Group_" data-name="&lt;Group&gt;">
```
Comment (SamChou19815) [inline assets/download-on-itunes.svg]: IDs nested inside svg can be a big problem. Also, the svgs are not minimized. Could you optimize the svg through tools like [svgo](https://github.com/svg/svgo)

**Comment by EmadBinAbid** [comment]
Sure, will do.

File: assets/download-on-itunes.svg (hunk 1 above)
Comment (EmadBinAbid) [reply assets/download-on-itunes.svg]: Sure, will do.

**Comment by SamChou19815** [comment]
The problem is already fixed in some other PRs landed a while ago.

**Pull request closed without merging**